benchmarks/TFLite/CMakeLists.txt

```
@@ -64,8 +64,22 @@ set(MOBILEBERT_FP32_MODULE
   TAGS
     "fp32"
   SOURCE
-    # Mirror of https://tfhub.dev/tensorflow/lite-model/mobilebert/1/default/1
-    "https://storage.googleapis.com/iree-model-artifacts/mobilebertsquad.tflite"
+    # Mirror of https://storage.googleapis.com/tfhub-lite-models/iree/lite-model/mobilebert/fp32/1.tflite
```

Isn't there a normal tfhub link (even if it's backed by this)? Looks like it's this one?
```suggestion
    # Mirror of https://tfhub.dev/iree/lite-model/mobilebert/fp32/1
```